client/landing/stepper/declarative-flow/import-hosted-site.ts

```diff
@@ -85,9 +85,15 @@ const importHostedSiteFlow: Flow = {
 
 					if (
 						depUrl.startsWith( 'http' ) ||
-						[ 'blogroll', 'ghost', 'tumblr', 'livejournal', 'movabletype', 'xanga' ].indexOf(
-							providedDependencies?.platform as ImporterMainPlatform
-						) !== -1
+						[
+							'blogroll',
+							'ghost',
+							'tumblr',
+							'livejournal',
+							'movabletype',
+							'xanga',
+							'substack',
```

Comment:
The check (or just the list) seems like a candidate for shared variable since it's repeated so many times.

Reply:
That's true. There was a lot of copy-pasting between the flows. Initially, it was only one onboarding flow; now, we have three.

I'll create a ticket for refactoring/cleaning up flows and replacing the deprecated stepper navigation `goToStep` method with `submit`.

Reply:
https://github.com/Automattic/wp-calypso/issues/84748